components/dashboard/craco.config.js

```diff
@@ -23,6 +23,15 @@ module.exports = {
                         cookie: process.env.GP_DEV_COOKIE,
                     },
                 },
+                "/public-api": {
+                    target: "https://" + process.env.GP_DEV_HOST,
+                    ws: true,
+                    headers: {
+                        host: process.env.GP_DEV_HOST,
+                        origin: "https://" + process.env.GP_DEV_HOST,
+                        cookie: process.env.GP_DEV_COOKIE,
+                    },
+                },
```

Comment:
**question**: Is this needed here? Not sure but I'll let someone from the `@gitpod-io/engineering-webapp`  take a look.

Reply:
Sure!!